src/assets/i18n/de.json

```diff
@@ -204,12 +204,19 @@
 		"giveBadge": "Badge vergeben",
 		"nameWarning": "Beachte: Der Name wird als Klartext im Badge gespeichert.",
 		"multipleBadgeAwardHeadline": "Badge an mehrere Lernende gleichzeitig vergeben",
-		"uploadListText": "Du kannst eine Liste von Empfänger:innen importieren, die den Badge gleichzeitig erhalten sollen. Die Datei muss folgendermaßen aussehen:",
+		"uploadListText": "Du kannst eine <span class='tw-font-bold'>Liste von Empfänger:innen importieren</span>, die den Badge gleichzeitig erhalten sollen. So geht`s:",
```

Comment:
Backtick is wrong in German, should be `'` ("So geht's" or "So gehts")

Reply:
Fixed it